### PR TITLE
[New Rule] Potential Sudo Privilege Escalation via CVE-2019-14287

### PR DESCRIPTION
**Pull request by Aegrah**
## Summary
This rule monitors for the execution of a suspicious sudo command that is leveraged in CVE-2019-14287 to escalate privileges to root. Sudo does not verify the presence of the designated user ID and proceeds to execute using a user ID that can be chosen arbitrarily. By using the sudo privileges, the command "sudo -u#-1" translates to an ID of 0, representing the root user. 

### Detection
**No FPs in telemetry in the last 365 days, nor any FPs in my own test stacks.**

```
process where host.os.type == "linux" and event.action == "exec" and event.type == "start" and 
process.name == "sudo" and process.args == "-u#-1"
```

image

### CODE REVIEW

File: rules/linux/privilege_escalation_sudo_cve_2019_14287.toml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+min_stack_version = "8.3.0"
+updated_date = "2023/08/30"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule monitors for the execution of a suspicious sudo command that is leveraged in CVE-2019-14287 to escalate 
+privileges to root. Sudo does not verify the presence of the designated user ID and proceeds to execute using a user ID
+that can be chosen arbitrarily. By using the sudo privileges, the command "sudo -u#-1" translates to an ID of 0, 
+representing the root user.
```
Comment (Aegrah) [inline rules/linux/privilege_escalation_sudo_cve_2019_14287.toml:15]:
```suggestion
This rule monitors for the execution of a suspicious sudo command that is leveraged in CVE-2019-14287 to escalate 
privileges to root. Sudo does not verify the presence of the designated user ID and proceeds to execute using a user ID
that can be chosen arbitrarily. By using the sudo privileges, the command "sudo -u#-1" translates to an ID of 0, 
representing the root user. This exploit may work for sudo versions prior to v1.28.
```

**Review by imays11 — APPROVED** [review]
nice!

File: rules/linux/privilege_escalation_sudo_cve_2019_14287.toml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Sudo Privilege Escalation via CVE-2019-14287"
+references = ["https://www.exploit-db.com/exploits/47502"]
+risk_score = 47
+rule_id = "8af5b42f-8d74-48c8-a8d0-6d14b4197288"
+severity = "medium"
+tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Privilege Escalation", "Data Source: Elastic Defend"]
```
Comment (w0rk3r) [inline rules/linux/privilege_escalation_sudo_cve_2019_14287.toml:26]:
```suggestion
tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Privilege Escalation", "Data Source: Elastic Defend", "Use Case: Vulnerability"]
```

**Review by w0rk3r — APPROVED** [review]
Nice one

**Pull request merged**